mappings/net/minecraft/state/property/IntProperty.mapping

```diff
@@ -7,11 +11,14 @@ CLASS net/minecraft/class_2758 net/minecraft/state/property/IntProperty
 	METHOD method_11867 of (Ljava/lang/String;II)Lnet/minecraft/class_2758;
 		COMMENT Creates an integer property.
 		COMMENT
-		COMMENT <p>{@code min} must be non-negative and {@code max} must be greater than {@code min}.
+		COMMENT <p>Note that this method takes {@code O(max - min)} time as it computes
```

Comment:
Should be `O(n)` as it's linear. Imo just mentioning that it computes all possible values suffices.